Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ElasticWindowMultiple.py

```diff
@@ -13,6 +13,12 @@
 import numpy as np
 
 
+def workspaces_have_same_size(workspaces):
+    first_size = len(workspaces[0].readY(0))
+    differently_sized_workspaces = [workspace for workspace in workspaces[1:] if len(workspace.readY(0)) != first_size]
+    return len(differently_sized_workspaces) == 0
```

Comment:
```suggestion
    return sum(1 for workspace in workspaces[1:] if len(workspace.readY(0)) != first_size) == 0
```
condensing the two lines into one, and it will be quicker as we're not copying over workspaces into the new list.
Although still use two lines if you think the single line is too complex

Reply:
👍 I will probably leave it as is to help readability